app/Http/Controllers/User/UsersController.php

```diff
@@ -271,15 +271,22 @@ private function loginWithEmail($email, $password)
         return $user;
     }
 
+
     private function loginWithSocialProvider($provider_id, $provider_user_id)
     {
-        $user = User::with('accounts')->whereHas('accounts', function ($query) use ($provider_id, $provider_user_id) {
-            $query->where('provider_id', $provider_id)->where('provider_user_id', $provider_user_id);
-        })->first();
+
+
+        $user = \DB::table('dbp_users.users')->join('dbp_users.user_accounts', function ($join) use ($provider_id, $provider_user_id) {
```

Comment:
On the database name please get the value from the configuration, this way the database names are handled by the environment files.

For example `\DB::table(config('database.connections.dbp_users.database') . '. users')` instead of `\DB::table('dbp_users.users')`